sos/plugins/virsh.py

```diff
@@ -43,8 +43,7 @@ def setup(self):
 
         # get network, pool and nwfilter elements
         for k in ['net', 'nwfilter', 'pool']:
-            self.add_cmd_output('%s %s-list' % (cmd, k))
-            k_list = self.get_command_output('%s %s-list' % (cmd, k))
+            k_list = self.get_cmd_output_now('%s %s-list' % (cmd, k))
```

Comment:
with `save=False`?

Reply:
Same as `pcp` - without `save=False` we "automatically" get the `add_cmd_output()` call from above.